config/cesm/machines/config_compilers.xml

```diff
@@ -601,6 +601,60 @@ using a fortran linker.
   </SLIBS>
 </compiler>
 
+<compiler MACH="coeus" COMPILER="gnu">
```

Comment:
The idea in config_compilers.xml is to define a base set of flags for the gnu compiler (above in the file) and then only modify those settings for a particular machine as necessary.   Can you reduce this xml clause to be only the flags that are changed for coeus from the default gnu settings?

Reply:
I'll work on seeing what the minimum settings to still have the regression tests pass to the same level are.